penguin/web/server.py

```
@@ -8,8 +8,11 @@
 import logging
 import os
 import sys
+from datetime import datetime, timezone
 from ipaddress import ip_address
-from typing import Optional, Sequence
+from logging.handlers import RotatingFileHandler
+from pathlib import Path
+from typing import Any, Optional, Sequence
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)